k8s/cmd/admission-webhook/const.go

```diff
@@ -12,8 +12,8 @@ const (
 	repoEnv                      = "REPO"
 	initContainerEnv             = "INITCONTAINER"
 	tagEnv                       = "TAG"
-	jaegerHostEnv                = "JAEGER_SERVICE_HOST"
-	jaegerPortEnv                = "JAEGER_SERVICE_PORT_JAEGER"
+	jaegerHostEnv                = "JAEGER_AGENT_HOST"
+	jaegerPortEnv                = "JAEGER_AGENT_PORT"
```

Comment:
Do we still need these constants?  Are they being used anywhere?

Reply:
We need to make the constant env name match the env name used by pkg/tools/tools.go::InitJaeger() so the values put in the initContainer patch are consumed properly.